### PR TITLE
Update license file and headers to GNU AGPLv3

### PR DESCRIPTION
**Pull request by AdrySky**
# Description

Update license file and headers to GNU Affero General Public License v3

Fixes # (issue)

## Type of change

- [ ] New Exercise
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Adding automation capabilities with GitHub Action

# Tested on?

- [x] Windows
- [ ] Linux Ubuntu
- [ ] Centos
- [ ] Mac
- [ ] Others  (State here -> xxx )

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

### CODE REVIEW

File: java-core/pom.xml [hunk 1]
```diff
@@ -1,19 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~  Copyright (c) 2020-2021 CertifAI Sdn. Bhd.
+  ~ #
+  ~ #    Copyright (c) 2020-2021 CertifAI Sdn. Bhd.
+  ~ #
+  ~ #    This program is part of OSRFramework. You can redistribute it and/or modify
```
[the remark this answers is not in the document]
Comment (AdrySky) [reply java-core/pom.xml:6]: Sure, I'll change it. I was trying to replicate the snippet u sent to me.

[the remark this answers is not in the document]
Comment (codenamewei) [reply java-core/pom.xml:6]: np...that was probably for python :)

**Comment by AdrySky** [comment]
@codenamewei I've change the license header as requested. Please help me to review it since there a failed checks.

**Comment by codenamewei** [comment]
> @codenamewei I've change the license header as requested. Please help me to review it since there a failed checks.

Thanks , checking it out.

**Comment by codenamewei** [comment]
@fariskamaludin-skymind 

i reran the job and its failing still
Screenshot 2021-06-11 at 8 11 29 AM


do you still have plans to separate the script to two ? 
1. for build
2. for sonar
3.

**Comment by fariskamaludin-skymind** [comment]
> @fariskamaludin-skymind
> 
> i reran the job and its failing still
> Screenshot 2021-06-11 at 8 11 29 AM
> 
> do you still have plans to separate the script to two ?
> 
> 1. for build
> 2. for sonar

@codenamewei I already split the jobs into two but it is only written in the same workflow. I think you reran the workflow prior my workflow update. I already triggered the new workflow by `review_requested` event as I set myself as a new reviewer. The PR have been updated with the new workflow's checks.

**Comment by codenamewei** [comment]
@fariskamaludin-skymind Thanks , its clear now ...i can see how build with maven is passing

**Review by codenamewei — APPROVED** [review]
looks all good now.

**Pull request merged**